.gitignore

```diff
@@ -0,0 +1 @@
+.vscode/
```

Comment:
It may sound really silly but I'm not sure we should be adding a .gitignore to the documentation repo.

Only because it has never been there and I'm afraid of changing things.